backend/gradle.lockfile

```diff
@@ -0,0 +1,309 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
@corneliusroemer can you explain to me how this file was generated? Just trying to fully understand this change